src/v/cloud_storage/remote_segment.h

```diff
@@ -414,7 +415,7 @@ class remote_segment_batch_reader final {
     storage::log_reader_config _config;
     partition_probe& _probe;
     ts_read_path_probe& _ts_probe;
-    ss::circular_buffer<model::record_batch> _ringbuf;
+    data_t _ringbuf;
```

Comment:
the variable should probably be renamed since it's no longer a ring buffer

Reply:
I agree!